Makefile

```diff
@@ -125,6 +125,11 @@ test: generate
 test-style:
 	@scripts/validate-go.sh
 
+.PHONY: validate-generated
+validate-generated: generate
+	@echo "==> Checking generated files <=="
+	! git diff --name-only pkg/ | grep _generated.go
```

Comment:
```suggestion
	git diff --name-only --exit-code pkg/ | grep _generated.go
```

Reply:
I don't think that will work actually. I'm using the `!` to invert the return code of the entire statement--so really it affects `grep`. I want to return 1 if `grep` matches rather than the default of 0.

Reply:
Got it thanks!